pre_processing/orac_preproc.F90

```diff
@@ -459,10 +459,11 @@ subroutine orac_preproc(mytask, ntasks, lower_bound, upper_bound, driver_path_fi
    preproc_opts%ecmwf_time_int_method     = 2
    preproc_opts%use_ecmwf_snow_and_ice    = .true.
    preproc_opts%use_modis_emis_in_rttov   = .false.
-   preproc_opts%ecmwf_path(2)             = ' '
-   preproc_opts%ecmwf_path2(2)            = ' '
-   preproc_opts%ecmwf_path3(2)            = ' '
-   preproc_opts%ecmwf_nlevels             = 0
+   preproc_opts%nwp_path(2)             = ' '
+   preproc_opts%nwp_path2(2)            = ' '
+   preproc_opts%nwp_path3(2)            = ' '
+   preproc_opts%nwp_nlevels             = 0
+   preproc_opts%nwp_time_factor           = 6.
```

Comment:
Shouldn't this be 6/24? That's how it was originally defined at L111 of set_ecmwf